CHANGELOG.next.toml

```diff
@@ -485,3 +485,9 @@ message = "Make generated enum `values()` functions callable in const contexts."
 references = ["smithy-rs#2011"]
 meta = { "breaking" = false, "tada" = false, "bug" = false, "target" = "all" }
 author = "lsr0"
+
+[[aws-sdk-rust]]
+message = "Normalize URI paths per RFC3986 when constructing canonical requests, except for S3."
+references = ["smithy-rs#2018"]
+meta = { "breaking" = true, "tada" = false, "bug" = false }
```

Comment:
What is breaking about this? Seems more like a bug fix?

Reply:
Oh my bad. The code prior to the PR wasn't working as intended, so there is nothing to break. It is a bug fix.